frontend/src/concepts/hardwareProfiles/HardwareProfileSelect.tsx

```diff
@@ -397,51 +400,61 @@ const HardwareProfileSelect: React.FC<HardwareProfileSelectProps> = ({
                 }
               >
                 <>
-                  <MenuGroup
-                    key="project-scoped"
-                    data-testid="project-scoped-hardware-profiles"
-                    label={
-                      <Flex
-                        spaceItems={{ default: 'spaceItemsXs' }}
-                        alignItems={{ default: 'alignItemsCenter' }}
-                        style={{ paddingBottom: '5px' }}
-                      >
-                        <FlexItem style={{ display: 'flex', paddingLeft: '12px' }}>
-                          <img
-                            style={{ height: '20px', paddingTop: '3px' }}
-                            src={typedObjectImage(ProjectObjectType.project)}
-                            alt=""
-                          />
-                        </FlexItem>
-                        <FlexItem>Project-scoped Hardware profiles</FlexItem>
-                      </Flex>
-                    }
-                  >
-                    {getHardwareProfiles()}
-                  </MenuGroup>
-
-                  <Divider component="li" />
-                  <MenuGroup
-                    key="global-scoped"
-                    data-testid="global-scoped-hardware-profiles"
-                    label={
-                      <Flex
-                        spaceItems={{ default: 'spaceItemsXs' }}
-                        alignItems={{ default: 'alignItemsCenter' }}
-                        style={{ paddingBottom: '5px' }}
-                      >
-                        <FlexItem
-                          style={{ display: 'flex', paddingLeft: '10px' }}
-                          data-testid="ds-project-image"
+                  {filteredHardwareProfiles.length > 0 && (
+                    <MenuGroup
+                      key="project-scoped"
+                      data-testid="project-scoped-hardware-profiles"
+                      label={
+                        <Flex
+                          spaceItems={{ default: 'spaceItemsXs' }}
+                          alignItems={{ default: 'alignItemsCenter' }}
+                          style={{ paddingBottom: '5px' }}
                         >
-                          <GlobalIcon />
-                        </FlexItem>
-                        <FlexItem>Global hardware profiles</FlexItem>
-                      </Flex>
-                    }
-                  >
-                    {getDashboardHardwareProfiles()}
-                  </MenuGroup>
+                          <FlexItem style={{ display: 'flex', paddingLeft: '12px' }}>
+                            <img
+                              style={{ height: '20px', paddingTop: '3px' }}
+                              src={typedObjectImage(ProjectObjectType.project)}
+                              alt=""
+                            />
+                          </FlexItem>
+                          <FlexItem>Project-scoped Hardware profiles</FlexItem>
```

Comment:
```suggestion
                          <FlexItem>Project-scoped hardware profiles</FlexItem>
```
according to mocks https://www.figma.com/proto/ZvdiudUctzI3HM8W2IHOuT/Project-Enhancements---2025?page-id=6502%3A12147&node-id=6502-13032&viewport=184%2C261%2C0.06&t=jb8nFHfY21A7nqQQ-1&scaling=scale-down-width&content-scaling=fixed&starting-point-node-id=6502%3A12148